### PR TITLE
style: model diagram gets a border

### PR DESCRIPTION
**Pull request by jamiewaese-uncharted**
# Description

* Include a summary of the changes and the related issue. 
* Include relevant motivation and context.

Resolves #(issue)

### CODE REVIEW

File: packages/client/hmi-client/src/page/project/tera-project.vue [hunk 1]
```diff
@@ -336,6 +336,7 @@ const loadingTabIndex = ref<number | null>(null);
 function setActiveTab() {
 	activeTabIndex.value = tabStore.getActiveTabIndex(projectContext.value);
 	loadingTabIndex.value = null;
+	console.log('projectContext.value', projectContext.value);
```
Comment (echl) [inline packages/client/hmi-client/src/page/project/tera-project.vue:339]: Can you remove this console.log

File: packages/client/hmi-client/src/page/project/tera-project.vue [hunk 2]
```diff
@@ -485,6 +486,34 @@ function formatAuthorTimestamp(username, timestamp) {
 	}
 	return `${username} on ${formatDdMmmYyyy(timestamp)}`;
 }
+/*
```
Comment (echl) [inline packages/client/hmi-client/src/page/project/tera-project.vue:489]: Can you delete the commented out stuff?

**Comment by echl** [comment]
Can you include video/screenshot of changes?

File: packages/client/hmi-client/src/page/project/tera-project.vue (hunk 2 above)
Comment (jamiewaese-uncharted) [reply packages/client/hmi-client/src/page/project/tera-project.vue]: oops. thanks.

File: packages/client/hmi-client/src/page/project/tera-project.vue [hunk 3]
```diff
@@ -499,6 +499,19 @@ function formatAuthorTimestamp(username, timestamp) {
 	}
 	return `${username} on ${formatDdMmmYyyy(timestamp)}`;
 }
+
+// if this is a fresh load, open the overview page
```
[the remark this answers is not in the document]
Comment (jamiewaese-uncharted) [reply packages/client/hmi-client/src/page/project/tera-project.vue]: done

**Review by shawnyama — APPROVED** [review]

**Pull request merged**